files/zh-cn/web/javascript/guide/regular_expressions/groups_and_backreferences/index.md

```diff
@@ -175,10 +175,10 @@ original_slug: Web/JavaScript/Guide/Regular_Expressions/Groups_and_Ranges
 
 ```js
 var aliceExcerpt = "There was a long silence after this, and Alice could only hear whispers now and then.";
-var regexpVowels = /[aeiouy]/g;
+var regexpVowels = /[aeiou]/g;
 
 console.log("Number of vowels:", aliceExcerpt.match(regexpVowels).length);
-// Number of vowels: 25
+// Number of vowels: 24
 ```
```

Comment:
Hi @5z5. We have removed this example in en-US. could you also made the same change here?

![image](https://user-images.githubusercontent.com/15844309/216763188-a080b082-9304-406d-9173-52b63aed6d1d.png)